### DISCUSSION TITLE
Lỗi FB.login() called before FB.init().

### DISCUSSION THREAD

**Discussion by tuanna-comartek** [cuongdevjs/reactjs-social-login#53] [General]
Mình đang dev bằng NextJs, khi mình sử dụng LoginWithFacebook thì bị lỗi : FB.login() called before FB.init(). Mình vẫn dùng được LoginWithGoogle bình thường, Mong được bạn giải đáp

**Comment by cuongdevjs** [comment]
Bạn có sử dụng sdk facebook trước đó chưa?

**Reply by cuongdevjs** [reply]
Cái này bạn phải đảm bảo không có 2 sdk chạy cùng lúc thì mới hoạt động được bạn nhé. Bạn phải tìm cách xóa sdk script trước khi load component Facebook nhé.
